src/device-state/current-state.ts

```diff
@@ -244,12 +228,19 @@ function reportCurrentState(attempts = 0) {
 						maxDelay,
 						// Start the polling at the value given by the API if any
 						e.retryAfter ?? MINIMUM_BACKOFF_DELAY,
+						'Device state report',
 					);
 				} else {
 					eventTracker.track('Device state report failure', {
 						error: e.message,
 					});
-					backoff(reportCurrentState, stateReportErrors++, maxDelay);
+					backoff(
+						reportCurrentState,
+						stateReportErrors++,
+						maxDelay,
+						MINIMUM_BACKOFF_DELAY,
+						'Device state report',
```

Comment:
I added this bit so the log message is more useful when used in more places

Reply:
see my comment below on how it maybe this could be a callback instead of a string